### ISSUE TITLE
<10.6.1> bedrock users are stuck at loading terrian when they tried to join server

### ISSUE THREAD

**Issue by adalmo** [casperge/RealisticSeasons-issues#218]
am not sure why its happening on bedrock users when i tried to use this plugin when i try to remove this one they can join as well maybe chunk issue or smh ig if it get fix it goes more good for bedrock players as well

**Comment by casperge** [comment]
Have you had any errors in your console at startup?

Also, can you try updating ProtocolLib to the latest dev build?

**Comment by adalmo** [comment]
yes i got latest dev of protocollib and no errors are showing on logs

**Comment by casperge** [comment]
What is your server, geyser version? Also, what type of server software are you using?
